deployment/helm/topology-aware/values.yaml

```diff
@@ -22,6 +35,9 @@ resources:
 nri:
   patchRuntimeConfig: false
 
+plugin:
+  enableTestAPIs: false
```

Comment:
Yeah, probably `plugin` is not a very good choice here. I'm not too keen on `config` because we already have the two config sections `policyConfig` and `commonConfig` (which I am not too fond of either). I pulled `plugin` for the name of this section mostly because it enables (any) extra test APIs that might be compiled into the plugin binary.

Maybe this should be instead something like `testAPIs.enable: true`, or then we could combine the config bits into a single one with 3 sections `config.policy`, `config.common`, and `config.test[APIs]` and then under `config.test` we'd have `enable: false`...

Reply:
Ah I missed that `commonConfig `and `policyConfig `is already there. Maybe it is actually fine for now to keep it as it is. 

But I also kind of like the idea of `config.policy`, `config.common`, and `config.test.....`

Reply:
@fmuyassarov I updated the helm charts and the e2e tests to use the above outlined helm chart template with config.{policy,common,test}. PTAL, if this looks/feels like a better fit. I am already past the point where I could really trust my judgement. I can barely see the forest from the trees any more.